fastpy/requirements.txt

```diff
@@ -0,0 +1,191 @@
+absl-py==0.6.1
```

Comment:
How come we have so many dependencies even though we only use stdlib stuff?

Reply:
I guess you used site-packages... We should remove them though since none of them are requirements for this project.

Reply:
Probably a bleed from another virtualenv I based on, will clean it up.